dangerfile.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-// import { danger, warn } from "danger"
+import { danger, warn, markdown } from "danger"
```

Comment:
This was commented out, but now it's not. I remember in #54 (https://github.com/danger/danger-js/commit/0ba01b15e56093c5ce11ecccccef738b3ef66cf0#diff-3abee5fb96982a8bed436d8b59267dd8R76) that the danger DSL was added to the global namespace for convenience, at least as I understood it. Is this still the case?